feg/gateway/services/csfb/servicers/test/csfb_test.go

```diff
@@ -295,7 +308,8 @@ func TestCsfbServer_MMEResetIndication(t *testing.T) {
 	client := protos.NewCSFBFedGWServiceClient(conn)
 	reply, err := client.MMEResetIndication(context.Background(), req)
 	assert.NoError(t, err)
-	assert.Equal(t, &orcprotos.Void{}, reply)
+	equal := proto.Equal(&orcprotos.Void{}, reply)
+	assert.True(t, equal)
```

Comment:
I'm assuming this pattern doesn't provide good error messages -- I believe there's a way to conditionally print the values that differ in the assert call?

Reply:
That is right. We optimized a little and introduced this function:

https://github.com/alexzurbonsen/magma/blob/d3ac7bfb8dc6e4857a98888bfbe3ea0401e57092/orc8r/cloud/go/test_utils/unit_test_proto_utils.go#L52-L67

Doesn't show preview, I guess since it is on a fork. Let me copy it here:
```
// AssertMessagesEqual compares two proto.Message's with proto.Equal.
// Prints string representations of messages upon inequality.
// Note, that string representations provided by protobuf are not stable.
// Please only pay attention to the field values themselves, not to whitespaces
// in between.
func AssertMessagesEqual(t *testing.T, expected, actual proto.Message) bool {
	equal := proto.Equal(expected, actual)
	if !equal {
		assert.Fail(t, fmt.Sprintf("Not equal: \n"+
			"expected: %s\n"+
			"actual: %s\n"+
			"Note, that string representations provided by protobuf are not guaranteed to be stable and the comparison between messages does not rely on them. Please pay attention to differences in the field values only.",
			expected.String(), actual.String()))
	}
	return true
}
```